src/components/ArticlePreview.jsx

```diff
@@ -46,7 +42,7 @@ export default class ArticlePreview extends BaseComponent {
           <p className="article-preview__content__teaser">{article.teaser}</p>
           <SharingButtons
             title={article.title}
-            url={'thegazelle.org' + url}
+            url={`thegazelle.org${url}`}
```

Comment:
@zanemountcastle this is kinda unrelated, but shouldn't we put a `https://` in front of this?